ci/release-info-v2.py

```diff
@@ -0,0 +1,96 @@
+import json
+import os
+import re
+import sys
+from datetime import date
+from subprocess import Popen, PIPE
+
+def read_version(crate):
+    # Reads package version from workspace manifest file
+    channel = Popen(
+        ['cargo', 'read-manifest', '--manifest-path', './{}/Cargo.toml'.format(crate)],
+        stdout=PIPE
+    )
+
+    # parse json data
+    data = json.load(channel.stdout)
+    version = data['version']
+    return version
+
+def check_version(ver):
+    # Checks package version for matching with the current tag reference
+    if ref is not None and ref != 'refs/tags/v' + str(ver):
```

Comment:
Better use formatting: `f'refs/tags/v{str(ver)}'`

Reply:
I'd prefer to use "string".format() of printf(), 'cause this improves readability and used in Python and C for many years, rather than use of such string formatting - this is relatively recent addition in Python 3 and doesn't have any advantages over format() or printf() functions, nor improves the readability. But here all of this is unnecessary - str() here is only for type cast the version into string in side cases, all other condition check expression remain as it was before in very first version of script.